### ISSUE TITLE
Is this plugin still working?

### ISSUE THREAD

**Issue by spkprav** [raphaelmerx/cordova-plugin-scheduler#1]
I tried integrating this plugin but doesn't work. I used the example code, I see the "scheduled job started" message in logs, but nothing happens.

but when I run using `adb shell cmd jobscheduler run -f <your.application.id> 999`, everything is fine. Any idea why Ionic doesn't start the service and has to be forced from adb shell?

**Comment by raphaelmerx** [comment]
In the job builder, we set https://github.com/raphaelmerx/cordova-plugin-scheduler/blob/8d525433e762c1ff249200985a1add151463f94e/src/android/SchedulerPlugin.java#L77

which means if there is no internet connectivity, the task will not be started. Could that be the issue?

**Comment by spkprav** [comment]
No, It isn't related to NetworkType, because the app requires internet to fetch data, so the Internet is always there. also, if I run using adb shell, it works perfect, it's just ionic not starting the service. 

My configuration:
```
Ionic:

   ionic (Ionic CLI)  : 4.12.0 (/Users/pk/.nvm/versions/node/v10.15.3/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.1.9

Cordova:

   cordova (Cordova CLI) : 8.1.2 (cordova-lib@8.1.1)
   Cordova Platforms     : android 7.1.4, ios 4.5.4
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 1.2.1, (and 16 other plugins)

System:

   Android SDK Tools : 26.1.1 (/Users/pk/Library/Android/sdk)
   NodeJS            : v10.15.3 (/Users/pk/.nvm/versions/node/v10.15.3/bin/node)
   npm               : 6.4.1
   OS                : macOS Mojave
   Xcode             : Xcode 10.2.1 Build version 10E1001
```

**Comment by Elyft** [comment]
i don't think that is the good idea to start the task if the user have an internet connection
Can you explain me why this choice? @raphaelmerx

**Comment by DrSiemer** [comment]
Looks like this plugin is dead? Similar issue here targeting Android 11. Simulating a background-fetch event via adb works, but only while the app is running. The scheduled event is never fired. The `shell dumpsys jobscheduler` command generates a wall of log that does not make much sense to me.

**Comment by DrSiemer** [comment]
The solution that worked for me after a lot of research and debugging is to add this permission request to the config.xml:

```
<config-file parent="./" target="app/src/main/AndroidManifest.xml" xmlns:android="http://schemas.android.com/apk/res/android">
  <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
</config-file>
```
